qemu/tests/cfg/qemu_guest_agent_hotplug.cfg

```diff
@@ -56,7 +56,12 @@
         gagent_stop_cmd = "net stop qemu-ga"
         gagent_status_cmd = sc query qemu-ga |findstr "RUNNING"
         gagent_pkg_check_cmd = wmic product get name |findstr "QEMU guest agent"
-        gagent_host_path = "/tmp/${qemu_ga_pkg}"
+        gagent_host_path = "/var/tmp/"
+        gagent_download_cmd = "wget https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/latest-qemu-ga/${qemu_ga_pkg} -O ${gagent_host_path}"
```

Comment:
Need to remove Line 52?

Reply:
Seems Line 52 still there, it is redundant with this line.

Reply:
> Seems Line 52 still there, it is redundant with this line.
sorry, maybe I force checkbranch to master and abort this part. 
what about now.